### DISCUSSION TITLE
ccf-validation-tools/Skin/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#273] [General]
# ccf-validation-tools/Skin/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Skin/comments/

**Comment by emquardokus** [comment]
Skin_v1.3_DRAFT
https://docs.google.com/spreadsheets/d/1q2tYQ_uNh5O_eLOMUZm64ncCUeJc8mrern3zkRX3Ppw/edit#gid=269383687

**Blank ontology ID missing parent**
1) Added perineurial cell CL:0002576 which was recently added to CL to fix both row 48 and 68
In row 48, the term perineurial cells without ontology ID has no parent that is from the CL ontology.
In row 68, the term Perineural cell without ontology ID has no parent that is from the CL ontology.

Added parent CT/1 as keratinocyte	CL:0000312 for the missing Keratinocyte (Onychocyte) Add to CL; this is a specialized keratinocyte of finger and toe nails. It does not exist in CL. @aleixpuigb

**Reply by aleixpuigb** [reply]
Thank you @emquardokus, I will add Onychocyte this week.

**Comment by emquardokus** [comment]
Skin_v1.3_DRAFT
https://docs.google.com/spreadsheets/d/1q2tYQ_uNh5O_eLOMUZm64ncCUeJc8mrern3zkRX3Ppw/edit#gid=269383687

**Different labels**
FIXED: In rows In row 24, and 52, the term CL:0000669 has different name/label in the source ontology. The name/label in the ASCT+B table is pericyte cell and the one in the ontology is pericyte. For reference, the given name/label by SMEs is Pericyte. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

FIXED: In row 50, the term CL:0000359 has different name/label in the source ontology. The name/label in the ASCT+B table is smooth muscle cell and the one in the ontology is vascular associated smooth muscle cell. For reference, the given name/label by SMEs is Perivascular smooth muscle cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
Skin_v1.3_DRAFT
https://docs.google.com/spreadsheets/d/1q2tYQ_uNh5O_eLOMUZm64ncCUeJc8mrern3zkRX3Ppw/edit#gid=269383687

**Different labels**


1. Removed period    In row 50, the term CL:0000359 has different name/label in the source ontology. The name/label in the ASCT+B table is vascular associated smooth muscle cell. and the one in the ontology is vascular associated smooth muscle cell. For reference, the given name/label by SMEs is Perivascular smooth muscle cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

2. Changed to correct CL ID for sebum secreting cell  CL:0000317    In row 24, the term CL:0000151 has different name/label in the source ontology. The name/label in the ASCT+B table is sebum secreting cell and the one in the ontology is secretory cell. For reference, the given name/label by SMEs is Germinative (epithelial) cell, Sebocyte. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by aleixpuigb** [comment]
## Skin_v1.3_DRAFT (05/10/2023)
https://docs.google.com/spreadsheets/d/1q2tYQ_uNh5O_eLOMUZm64ncCUeJc8mrern3zkRX3Ppw/edit#gid=269383687

### New cell type:
- Onychocyte (CL:4033056) will be present in the next CL release in two weeks.

### There are few cell types that can be mapped in the tables:
- Row 13: keratinocyte (Granular) in CL: ‘granular cell of epidermis’ (CL:0002189) is a specific subclass of keratinocyte in the stratum granulosum.
- Row 28: skin of epidermis has a subclass ‘skin of dermis’ (CL:0002551) which is more appropriate as the last AS is dermis.
- According to the markers of this publication PMID:[28926543](https://pubmed.ncbi.nlm.nih.gov/28926543) these cell types could have a dermal specific type, but without the appropriate label. They need to be confirmed:
      - Row 30: Dendritic cell (DC1) might be ‘langerin-positive dermal dendritic cell’ (CL:0002458) 
      - Row 31: Dendritic cell (DC2) might be ‘langerin-negative dermal dendritic cell’ (CL:0002459)
      - Row 32: Dendritic cell (monocyte-derived) might be ‘CD14-positive dermal dendritic cell’ (CL:0002526)
- Row 49: endothelial cell has a subclass more specific ‘dermis blood vessel endothelial cell’ (CL:2000010)
- Row 54: ‘endothelial cell of lymphatic vessel’ has a more specific subclass ‘dermis lymphatic vessel endothelial cell’ (CL:2000011)


### Repeated row: 
- Row 52 and 53 are repeated.

### Some cell types that can be added to the table:
- clear cell of eccrine sweat gland(CL:0002146)
- dark cell of eccrine sweat gland (CL:0002142)

**Comment by aleixpuigb** [comment]
**New term**
Skin Row 22 : onychocyte (CL:4033056)

**Comment by emquardokus** [comment]
@anitacaron   ASCT+B Validation Reports for Skin (2024-06-19) is pointing to a DOI'd v1.3 skin table  https://docs.google.com/spreadsheets/d/1q2tYQ_uNh5O_eLOMUZm64ncCUeJc8mrern3zkRX3Ppw/edit?gid=269383687#gid=269383687&range=56:56 rather than the v1.4_DRAFT table.

Skin_v1.4_DRAFT https://docs.google.com/spreadsheets/d/1_xLnPyGyQy9fae7DrWkeyEN-iLTP8JQMxTL7HHkxG6o/edit#gid=269383687

Found a Different label, but it did not show up in the validation report for either table. 

FIXED: ROW 70 Perivascular smooth muscle cell	smooth muscle cell	CL:0000359 to match ontology ID vascular associated smooth muscle cell	CL:0000359

Also FIXED: The term CL:0002521 has a different name/label in the source ontology in the following 1 row 56. The name/label in the ASCT+B table is subcutaneous fat cell and the one in the ontology is subcutaneous adipocyte. For reference, the given name/label by SMEs is Adipocyte. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
**Skin_v1.4_DRAFT**
https://docs.google.com/spreadsheets/d/1_xLnPyGyQy9fae7DrWkeyEN-iLTP8JQMxTL7HHkxG6o/edit?gid=269383687#gid=269383687

**Miscellaneous updates**
1. Update Ref/n/DOI to Ref/n/ID
2. Update format of all references
3. Fix first Ref field from Ref/2 to Ref 1

**Different labels**

1.  FIXED; The term CL:0000669 has a different name/label in the source ontology in the following 1 row 66. The name/label in the ASCT+B table is pericyte cell and the one in the ontology is pericyte. For reference, the given name/label by SMEs is Pericyte. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

 2.Change uberon to match label: arrector pili muscle of vibrissa	UBERON:0011940. The term UBERON:0002033 has a different name/label in the source ontology in the following 1 row 31. The name/label in the ASCT+B table is arrector pili muscle of vibrissa and the one in the ontology is arrector muscle of hair. For reference, the given name/label by SMEs is Arrector pili muscle. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

3. Fix using correct uberon id: hair follicle bulge	UBERON:0005975. The term UBERON:0002073 has a different name/label in the source ontology in the following 1 row 34. The name/label in the ASCT+B table is hair follicle bulge and the one in the ontology is hair follicle. For reference, the given name/label by SMEs is hair bulge. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.
